scripts/make_release_notes.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Converts github flavored markdown changelogs to release notes.
+"""
+
+import argparse
+import re
+import subprocess
+
+
+PRODUCTS = {
+    'Firebase/Auth/CHANGELOG.md': '{{auth}}',
+    'Firebase/Core/CHANGELOG.md': '{{core}}',
```

Comment:
Core's changelog goes at the top without a title. Perhaps a blank title?

Reply:
Done.